README.md

```diff
@@ -53,6 +53,10 @@ DFE_SIGN_IN_ISSUER=https://pp-oidc.signin.education.gov.uk:443
 DFE_SIGN_IN_REDIRECT_BASE_URL=https://localhost:3000
 DFE_SIGN_IN_IDENTIFIER=<paste identifier>
 DFE_SIGN_IN_SECRET=<paste secret>
+DFE_SIGN_IN_API_CLIENT_ID=teacherpayments
+DFE_SIGN_IN_API_SECRET=<paste secret>
+DFE_SIGN_IN_API_ENDPOINT=https://pp-api.signin.education.gov.uk
+DFE_SIGN_IN_ROLE=<paste role>
```

Comment:
Given that we know we are dealing with a single "Access to Teacher Payments " role for admin access to the app, how do you feel about simply defining it as a constant somewhere appropriate in the application rather than having it as something that is configured with an environment variable?  Perhaps in the auth controller, e.g. `Admin::AuthController::DFE_SIGN_IN_ADMIN_ROLE_CODE`. Also, We should use the `"code"` as DfE Sign-in team have said the code stays constant across all environments. In our case, the code is `"teacher_payments_access"`.